tsrc/cli/__init__.py

```diff
@@ -138,6 +139,18 @@ def resolve_repos(
     else:
         repos = repos_from_config(manifest, workspace.config)
 
+    if singular_remote:
+        filtered_repos = []
+        for repo in repos:
+            remotes = [
+                remote
+                for remote in repo.remotes
+                if re.search(singular_remote, remote.name)
```

Comment:
should this really be a regex? what if several remotes matches the regex. `origin` and `upstream` both match `.*r.*` for instance

Reply:
Good point @dmerejkowsky. Do you want me to change it so it matches exactly the name of the origin?

Reply:
@dmerejkowsky is there any other point to be addressed in this PR?

Reply:
Hi @vonpupp! Let me help you get this merged. :) Yes, I think we should only allow exact matches, case-sensitive (as git remotes are case-sensitive - I tested that). The reasoning is that we haven't promised regexp support in this feature docs and if we would add it now it could cause unexpected behavior for the users.